website/content/docs/install/index.mdx

```diff
@@ -145,54 +163,140 @@ $ sudo yum -y install nomad
 [Vault](/vault/tutorials) with the same command.
 
 </Tab>
-<Tab heading="Homebrew (macOS)">
+</Tabs>
 
-[Homebrew](https://brew.sh) is a free and open source package management system
-for Mac OS X. Install the official [Nomad
-formula](https://github.com/hashicorp/homebrew-tap) from the terminal.
+## Post-installation steps
 
-First, install the HashiCorp tap, a repository of all of the HashiCorp Homebrew
-packages.
+These steps are considered optional but can be helpful for running Nomad and to
+take advantage of additional Nomad functionalities.
+
+<Tabs>
+<Tab heading="Manual installation" group="manual">
+
+<Tabs>
+<Tab heading="Linux">
```

Comment:
```suggestion
<Tab heading="Linux Packages">
```

Matching the name with the tab above. The tutorial uses just `Linux` for this tab, but looking at it again I think `Linux Packages` is more accurate.